go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/coreos/go-systemd/v22
 
-go 1.12
+go 1.22
```

Comment:
This seem unintentional. The commit message does not explain the bump and the test baseline you have set is still go 1.12?

Reply:
This was unintended; removed.